### ISSUE TITLE
Some questions on how to set up and upgrade a new platform, based on a default one

### ISSUE THREAD

**Issue by santo74** [omega8cc/boa#512]
What's the recommended way to do the following ?

Set up a platform based on a default provided one, but with additional modules and still being able to easily upgrade this platform whenever a module or platform upgrade becomes available ?

E.g. I want to have a panopoly platform based on the one that gets installed by Octopus (i.e. panopoly with the additional optimizations). 
But I need additional modules, such as multilingual capabilities and other contributed modules.

1) should I create a new platform for this, based on the default one or should I just use the default one and add the additional modules to it directly (with drush dl <mod-name> or similar) ?

2) In case of the former, then what's the recommended way to do this ?
I can't find any makefile for the default panopoly platform installed by octopus (i.e. with optimizations)
and I'm unable to create a makefile of it, as explained in this issue: [https://github.com/omega8cc/boa/issues/510](https://github.com/omega8cc/boa/issues/510)

3) Depending on 1) and 2) above, how can I update the platform and/or the modules?
Can I run octopus up-stable all and will it automatically take care of upgrading/migrating the existing platform, modules and sites on the platform ?

I'm not new to drupal, but I'm quite new to Aegir/Boa and still trying to understand how everything works and how things have to be done.

I'm aware of the nice documentation on the website of omega8.cc, but reading through those docs multiple times, I still ddin't find a complete answer to my questions above.

**Comment by omega8cc** [comment]
We should improve our docs, perhaps.

**Comment by kopeboy** [comment]
Yes you should, cause I am lost exactly like santo74 or even more.

I would add this too: how to update a pre-built platform?
Where/how to download/use pre-built platforms when they are erased from my instance?
I can only see the platforms I am already using because all the others have been deleted after 30 days..

**Comment by omega8cc** [comment]
> How to update a pre-built platform?

You can't and shouldn't "update" any built-in platform.

Please read for more information and context: https://omega8.cc/your-drupal-site-upgrade-safe-workflow-298

You can add/install/re-add new platforms by running octopus upgrade also when there is no new BOA version if old platforms have been deleted, but it is not possible on our hosted BOA - you need to submit support requests then.

As for the BOA system and platforms upgrade - it is explained in this project docs, but it doesn't apply to hosted BOA, so it is not explained on our website, because *we* handle these upgrades as a part of managed hosting service.

**Comment by ryanhanau** [comment]
I agree that updating the help docs would be good. Please add specific examples.
